lib/paper_trail/reifier.rb

```diff
@@ -0,0 +1,256 @@
+module PaperTrail
+
+  # Given a version record and some options, builds a new model object.
+  # @api private
+  class Reifier
+
+    # @api private
+    def initialize(version, options)
+      @options = options.dup
+      @version = version
+    end
+
+    # See `VersionConcern#reify` for documentation.
+    # @api private
```

Comment:
No, it's actually public, but we're reserving the right to make breaking changes to it at any time.  

Maybe the words public and private are not the best choices, but that's what other major ruby libraries use, including  RSpec (https://github.com/rspec/rspec-expectations/blob/14faeab88f319ac0c2e4d793ec02c2b69eb52a5c/lib/rspec/matchers/composable.rb#L9) and Rails (https://github.com/rails/rails/blob/f454ad37ed46e14eb535eb3305d2c3183a5d52f7/activerecord/lib/active_record/reflection.rb#L68).